src/Reflection/DataTransferObjectProperty.php

```diff
@@ -150,4 +151,15 @@ private function resolveCasterFromDefaults(): ?Caster
 
         return null;
     }
+
+    private function resolveMappedProperty(): string | int
+    {
+        $attributes = $this->reflectionProperty->getAttributes(MapFrom::class);
+
+        if (! count($attributes)) {
+            return $this->reflectionProperty->name;
+        }
+
+        return $attributes[0]->getArguments()[0];
```

Comment:
use `$attributes[0]->newInstance()->name`

Reply:
Fixed in 6fd88a379b34965ea3c9ea687b84e4fc09a213c9.